### DISCUSSION TITLE
USB tethering

### DISCUSSION THREAD

**Discussion by RetoHB** [tesla-android/issue-tracker#117] [Q&A]
Is it possible to use USB tethering with the 44.2 release?
I connected the RPi to my Android phone via USB and enabled USB tethering in the settings. But I am not able to access Internet from Tesla Android...

**Comment by thisLambo** [comment]
I was able to get a hotspot working with my iPhone by pairing the phone with Tesla Android via Bluetooth and then in Connected Device Settings set the phone as "Internet Use". This triggered the hotspot on my phone.

Caveat being it's using bluetooth for internet traffic, but it worked.

The Carlinkit adapter may turn off bluetooth on Android. I had some oddities of it dropping and Bluetooth only turning back on after a reboot. 

YMMV

**Comment by RetoHB** [comment]
Indeed this seems to work with an Android phone as well!
However I am experiencing the same issue with bluetooth turning off and a reboot needed. However I do not have a Carlinkit adapter. @mikegapinski do you use bluetooth in any way for screen or audio capture? Or do you have an idea what the cause for this issue might be?

**Reply by mikegapinski** [reply]
no I don't. I have not done anything with bluetooth on the pi. Maybe it'll work better in the Android 13 update that I am working on

**Reply by mikegapinski** [reply]
Fixes for Bluetooth stability are on the way to 2022.44.3 :)

**Reply by RetoHB** [reply]
Nice!!! Just another 4 weeks of wait. ;-)

**Reply by mikegapinski** [reply]
> Nice!!! Just another 4 weeks of wait. ;-)

Lol, 44.3 means third release on week 44... so it'll come this week :)

**Reply by RetoHB** [reply]
Oh sure! I thought we are in week 40 only. ;-) Is it possible to upgrade between minor releases without scraping everything?
